compose.go

```diff
@@ -0,0 +1,213 @@
+package testcontainers
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	envProjectName = "COMPOSE_PROJECT_NAME"
+	envComposeFile = "COMPOSE_FILE"
+)
+
+// DockerCompose defines the contract for running Docker Compose
+type DockerCompose interface {
+	Down() ExecError
+	Invoke() ExecError
+	WithCommand([]string) DockerCompose
+	WithEnv(map[string]string) DockerCompose
+}
+
+// LocalDockerCompose represents a Docker Compose execution using local binary
+// docker-compose or docker-compose.exe, depending on the underlying platform
+type LocalDockerCompose struct {
```

Comment:
would it be possible to add a usage example to the README? I like that I can read through the tests for examples, but I think our README is lacking features in general